src/main/routes/course.js

```diff
@@ -109,9 +110,25 @@ router.route('/')
 	.get(html.auth_wrapper(async (req, res, next) => {
 		res.render('base_template', {
 			title: 'Course Portfolios',
-			body: mustache.render('course/index')
+			body: mustache.render('course/index', {added: ''})
```

Comment:
If the value passed to the renderer is always the same, why is it passed to the renderer?

Reply:
Nevermind, I see now